From 517dfb59019f425d684348566b738ab741893f60 Mon Sep 17 00:00:00 2001
From: Rob Crittenden <rcritten@redhat.com>
Date: Fri, 8 Feb 2013 10:08:36 -0500
Subject: [PATCH] Prevent a crash when no entries are successfully migrated.

It would fail in _update_default_group() because migrate_cnt wasn't
defined in context.

https://fedorahosted.org/freeipa/ticket/3386
---
 ipalib/plugins/migration.py |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/ipalib/plugins/migration.py b/ipalib/plugins/migration.py
index ccb484e4ae46826a29ffecae472a20198cadeb07..2fa9c3729f294c46c8d3784078b44c3173fc540e 100644
--- a/ipalib/plugins/migration.py
+++ b/ipalib/plugins/migration.py
@@ -749,6 +749,7 @@ can use their Kerberos accounts.''')
 
             valid_gids = []
             invalid_gids = []
+            context['migrate_cnt'] = 0
             migrate_cnt = 0
             for (dn, entry_attrs) in entries:
                 context['migrate_cnt'] = migrate_cnt
-- 
1.7.1

